Resources/Prototypes/Entities/Objects/Weapons/Guns/Basic/wands.yml

```
@@ -56,6 +62,10 @@
     proto: ProjectileFireball
     capacity: 5
     count: 5
+  - type: Tag
+    tags:
+    - Fireballwand
+    - WizardWand
```

Needs harmony comment
```suggestion
  # Harmony start
  - type: Tag
    tags:
    - Fireballwand
    - WizardWand
  # Harmony end
```